### PR TITLE
13389 load qr code in svg to preserve image quality

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#13389

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/DigitalCredentials/RegisterWallet.vue [hunk 1]
```diff
@@ -19,6 +19,7 @@
             <v-col cols="12" lg="3" class="px-2">
               <!-- Placeholder Url -->
               <QrcodeVue
+                render-as="svg"
```
Comment (severinbeauvais) [inline src/components/DigitalCredentials/RegisterWallet.vue:22]: Please attach a screenshot showing that this does what you intend.

**Comment by severinbeauvais** [comment]
Should this go to Test today and Prod on Thursday, or can it wait until next week?

**Comment by vysakh-menon-aot** [comment]
> Should this go to Test today and Prod on Thursday, or can it wait until next week?

Its not configured in Test or Prod. So not required.

File: src/components/DigitalCredentials/RegisterWallet.vue (hunk 1 above)
Comment (vysakh-menon-aot) [reply src/components/DigitalCredentials/RegisterWallet.vue:22]: Old:
image

New:
image

Comment (severinbeauvais) [reply src/components/DigitalCredentials/RegisterWallet.vue:22]: Oh, that's an awesome improvement! 👍

**Review by severinbeauvais — APPROVED** [review]

File: src/components/DigitalCredentials/DigitalWalletDownload.vue [hunk 2]
```diff
@@ -93,7 +93,7 @@
 
                   <p>2. Open the Trinsic Wallet and tap on the menu icon (<v-icon color="black">mdi-menu</v-icon>) on
                     the top left corner.</p>
-                  <p>3. Tap on Network and select "Sovrin Staging"</p>
+                  <p>3. Tap on Network and select "BCovrin Test"</p>
```
Comment (cameron-freshworks) [inline src/components/DigitalCredentials/DigitalWalletDownload.vue:96]: Thank you!

**Review by cameron-freshworks — APPROVED** [review]
Looks great, thanks!

**Pull request merged**